### DISCUSSION TITLE
How to customize Incubator.Textfield via ThemeManager?

### DISCUSSION THREAD

**Discussion by borstelmannl** [wix/react-native-ui-lib#2287] [Q&A] [answered]
Hey there,

how do i customize the IncubatorTextfield via the ThemeManager or even the normal Textfield?

```
ThemeManager.setComponentTheme("Textfield", {
    fieldStyle: {
      borderTopColor: "red",
      borderTopWidth: 2,
      borderLeftWidth: 2,
    },
    containerStyle: {
      backgroundColor: "red",
    },
  });
```

```
ThemeManager.setComponentTheme("Incubator.Textfield", {
    fieldStyle: {
      borderTopColor: "red",
      borderTopWidth: 2,
      borderLeftWidth: 2,
    },
    containerStyle: {
      backgroundColor: "red",
    },
  });
```
both or not working

**Comment by awssam** [comment, marked as answer]
i dont know if u have found the solution but for anyone who still looking how to set a default theme for TextField component here is it

    
    ThemeManager.setComponentTheme(
        'Incubator.TextField',
        {
            fieldStyle: {
                borderTopColor: "red",
                borderTopWidth: 2,
                borderLeftWidth: 2,
            },
            containerStyle: {
                backgroundColor: "red",
            },
        }
    );

for your case you just had to type TextField (pascal case) not Textfield
